core-web/libs/portlets/dot-ema/src/lib/components/edit-ema-navigation-bar/edit-ema-navigation-bar.component.scss

```diff
@@ -0,0 +1,70 @@
+@use "variables" as *;
+
+:host {
+    width: 80px;
```

Comment:
dont use variables in width/borders/etc ?

Reply:
We use variables when we are expecting to reuse it but this can be `rem` values tho